geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/StringBytesGlossary.java

```diff
@@ -0,0 +1,210 @@
+  @MakeImmutable
+  public static final byte[] bCOUNT = stringToBytes("COUNT");
+
+  // PubSubExecutor
+  @MakeImmutable
+  public static final byte[] bCHANNELS = stringToBytes("CHANNELS");
+
+  // InfoExecutor
+  @MakeImmutable
+  public static final byte[] bSERVER = stringToBytes("SERVER");
+  @MakeImmutable
+  public static final byte[] bCLUSTER = stringToBytes("CLUSTER");
+  @MakeImmutable
+  public static final byte[] bPERSISTENCE = stringToBytes("PERSISTENCE");
+  @MakeImmutable
+  public static final byte[] bREPLICATION = stringToBytes("REPLICATION");
+  @MakeImmutable
+  public static final byte[] bSTATS = stringToBytes("STATS");
+  @MakeImmutable
+  public static final byte[] bCLIENTS = stringToBytes("CLIENTS");
+  @MakeImmutable
+  public static final byte[] bMEMORY = stringToBytes("MEMORY");
+  @MakeImmutable
+  public static final byte[] bKEYSPACE = stringToBytes("KEYSPACE");
+  @MakeImmutable
+  public static final byte[] bDEFAULT = stringToBytes("DEFAULT");
+  @MakeImmutable
+  public static final byte[] bALL = stringToBytes("ALL");
+
+  // SlowlogExecutor and SlowlogParameterRequirements
+  @MakeImmutable
+  public static final byte[] bGET = stringToBytes("GET");
+  @MakeImmutable
+  public static final byte[] bLEN = stringToBytes("LEN");
+  @MakeImmutable
+  public static final byte[] bRESET = stringToBytes("RESET");
+
+  // ZAddExecutor and SetExecutor
+  @MakeImmutable
+  public static final byte[] bNX = stringToBytes("NX");
+  @MakeImmutable
+  public static final byte[] bXX = stringToBytes("XX");
+
+  // ZAddExecutor
+  @MakeImmutable
+  public static final byte[] bCH = stringToBytes("CH");
+  @MakeImmutable
+  public static final byte[] bINCR = stringToBytes("INCR");
+
+  // SetExecutor
+  @MakeImmutable
+  public static final byte[] bEX = stringToBytes("EX");
+  @MakeImmutable
+  public static final byte[] bPX = stringToBytes("PX");
+
+  // ********** Constants for Double Infinity comparisons **********
+  public static final String P_INF = "+inf";
+  public static final String INF = "inf";
+  public static final String P_INFINITY = "+Infinity";
+  public static final String INFINITY = "Infinity";
+  public static final String N_INF = "-inf";
+  public static final String N_INFINITY = "-Infinity";
+
+  @MakeImmutable
+  public static final byte[] bP_INF = stringToBytes(P_INF);
+
+  @MakeImmutable
+  public static final byte[] bINF = stringToBytes(INF);
+
+  @MakeImmutable
+  public static final byte[] bP_INFINITY = stringToBytes(P_INFINITY);
+
+  @MakeImmutable
+  public static final byte[] bINFINITY = stringToBytes(INFINITY);
+
+  @MakeImmutable
+  public static final byte[] bN_INF = stringToBytes(N_INF);
+
+  @MakeImmutable
+  public static final byte[] bN_INFINITY = stringToBytes(N_INFINITY);
+
+
+  // ********** Miscellaneous constants for convenience **********
+  /**
+   * byte value of the number 1
+   */
+  public static final byte NUMBER_0_BYTE = 48; // '0'
```

Comment:
This comment is inaccurate.

Reply:
Nice catch. The perils of copy/pasting...